pubspec.yaml

```diff
@@ -23,12 +23,16 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  auto_size_text: ^2.1.0
+  cupertino_icons: ^1.0.0
   firebase_auth: ^1.0.1
   firebase_core: ^1.0.1
   firebase_crashlytics: ^1.0.0
-  flutter_facebook_login: null
+  flutter_facebook_login: ^3.0.0
   flutter_switch: ^0.1.6-beta.1
-  google_sign_in: null
+  font_awesome_flutter: ^9.0.0
+  google_fonts: ^2.0.0
+  google_sign_in: ^5.0.1
   hive: ^2.0.0
   hive_flutter: ^1.0.0
   google_fonts: ^2.0.0
```

Comment:
google_fonts has been repeated in the pubspec.yaml file. Kindly remove this line.